core/lib/dal/src/proof_generation_dal.rs

```diff
@@ -1,4 +1,3 @@
-#![doc = include_str!("../doc/ProofGenerationDal.md")]
```

Comment:
The line is still removed.

Reply:
[Fixed](https://github.com/matter-labs/zksync-era/pull/2474/commits/aa9d3639050bf2a323c854a9c831750b26146eeb). ✅